### DISCUSSION TITLE
brackets inside other brackets?

### DISCUSSION THREAD

**Discussion by gamer2key** [adieyal/sd-dynamic-prompts#806] [Q&A] [answered]
It would be extremely helpful for me to make prompts like this:
```
{opt1{opt1.1|opt1.2}|opt2{opt2.1|opt2.2}}
```
Unfortunately putting brackets inside other brackets doesn't work with the default settings, is there any way to make it work? That would be very helpful

**Comment by akx** [comment, marked as answer]
What's the intent of a prompt like that? Nested brackets should work fine by default, provided the rest of the syntax is right...

**Reply by gamer2key** [reply]
Simply add more variations in a batch, probably though there are better ways that I ignore.
I just tried again and it actually works, probably the error was caused by the rest of the syntax, stupid me, I apologize for the trouble thank you!
